src/test/scala/temple/builder/DatabaseBuilderTest.scala

```diff
@@ -90,9 +95,12 @@ class DatabaseBuilderTest extends FlatSpec with Matchers {
       "test_service",
       BuilderTestData.sampleService.attributes,
       Set(CRUD.List),
+      isStruct = false,
       Metadata.Readable.This,
     )
     queries.keys should contain(CRUD.List)
     queries(CRUD.List) shouldBe DatabaseBuilderTestData.sampleListStatementEnumerateByCreator
   }
+
+  // TODO: test the list endpoint with isStruct=true
```

Comment:
This would actually be quite good to do since it tests the disjunction logic. Could you add one please

Reply:
I’ve realised this will never happen anyway, and so removed that disjunction function: a struct can never have createdBy, so it’s one or the other, but not both